dropwizard-auth/src/main/java/io/dropwizard/auth/Demo.java

```diff
@@ -0,0 +1,38 @@
+import javax.crypto.Cipher;
```

Comment:
The file has increased in 38 lines.
<!-- lookout footnote separator -->
_Comment made by the analyzer Dummy._